src/YoTennis/Models/GameModel.cs

```diff
@@ -9,22 +9,66 @@ namespace YoTennis.Models
     public class GameModel
     {
         public List<GameEvent> Events { get; private set; }
-        public State CurrentState { get; private set; }
+        public State CurrentState { get; private set; } = new State();
 
-        public void AddPoint(Player GetPointPlayer, PointKind PointKind, ServeSpeed ServeSpeed)
+        public void AddPoint(Player getPointPlayer)
         {
-            //AddPoint
-            /*
-            if (GetPointPlayer == Player.First)
+            var score = CurrentState.ScoreInGame.Score = CurrentState.ScoreInGame.Score + PlayerScore.ForPlayer(getPointPlayer, 1);
+            if (score.FirstPlayer >= 4 && score.SecondPlayer + 1 < score.FirstPlayer)
```

Comment:
minor optimization: since you already know who's got a point, than you can test only for that player.